app/src/pages/inside/testCaseLibraryPage/expandedOptions/expandedOptions.tsx

```diff
@@ -36,13 +39,26 @@ export const ExpandedOptions = () => {
   const [activeFolder, setActiveFolder] = useState(null);
   const [isEmptyFolder, setIsEmptyFolder] = useState(false);
   const { formatMessage } = useIntl();
+  const dispatch = useDispatch();
 
   const { filteredTestCases, loading, hasTestCases, searchValue, setSearchValue } = useTestCases();
 
   const setAllTestCases = () => {
     setActiveFolder(null);
   };
 
+  const showCreateFolderModal = () => {
+    dispatch(
+      showModalAction({
+        id: 'CreateFolderModal',
+        data: {
+          areFoldersPresent: !!FOLDERS.length,
```

Comment:
```suggestion
          areFoldersPresent: Boolean(FOLDERS.length),
```

Reply:
Let's use `!isEmpty(FOLDERS)` from ldoash for consistency